### ISSUE TITLE
As a venue owner, I want to be able to update information about my venue.

### ISSUE THREAD

**Issue by Tris10N** [wsu-cpts489-fa20/bp-music#4]
- [ ] Update venue description
- [ ] Update location
- [ ] Update contact information
- [ ] Change password

Needed so venue profiles can change their public and private account information

**Comment by Tris10N** [comment]
Moving to milestone 3.  Was blocked on login functionality, bugs, and higher priority work that needed to be completed this milestone.  Thus, it has been deferred.
